pkgs/top-level/all-packages.nix

```diff
@@ -19285,6 +19306,7 @@ in
 
   # Intentionally lacks recurseIntoAttrs, as -rc kernels will quite likely break out-of-tree modules and cause failed Hydra builds.
   linuxPackages_testing = linuxPackagesFor pkgs.linux_testing;
+  linuxPackages_next = args: linuxPackagesFor (pkgs.linux_next args);
```

Comment:
I am a bit hesitant to have more kernel to test when somebody updates kernel modules.

Reply:
cc @NeQuissimus